src/main/AndroidManifest.xml

```diff
@@ -20,82 +20,85 @@ You should have received a copy of the GNU General Public License along with Sim
 @copyright 2015 Vojtech Kral
 -->
 <!-- package name and version codes are overwritten by Gradle -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+<manifest
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    package="nl.mpcjanssen.simpletask" >
-
+    package="nl.mpcjanssen.simpletask"
+    >
```

Comment:
Modified manifest to use the same xml whitespace layout as everything else. Everything is cosmetic unless noted otherwise.